hw/top_earlgrey/data/top_earlgrey_pkg.sv.tpl

```diff
@@ -0,0 +1,36 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+<%
+import re
+import topgen.lib as lib
+
+def camelcase(str):
```

Comment:
@lenary
I picked this from the C header template which I presume you added. LMK if you have any ideas on how we can share this (perhaps move this to topgen.py instead?).

Reply:
We can probably move it to `topgen.lib` in the short term. In the long term, the function is really brittle and I'd prefer an easier way of doing the same thing, but that would be a lot more complex I think. I'm happy my answer did not block this PR.

Reply:
Sounds good, thanks.